### DISCUSSION TITLE
[Announcement] Welcome Sungjun Kim to the BentoML core team!

### DISCUSSION THREAD

**Discussion by parano** [bentoml/BentoML#1022] [General]
Hi BentoML community,

I'm thrilled to announce that Sungjun Kim(@withsmilo) will be joining the BentoML open-source team as the fifth core team member, helping to build and maintain the project! 

Sungjun Kim is the Architect and team lead at LINE Plus Crop's machine learning platform team. He has been the maintainer of the model serving project Clipper for the past two years and started contributing to BentoML since Sep 2019. He has been very active in the community, helping other community members, contributing high-quality code, and providing valuable feedback for the BentoML team. He has also been leading the effort of building the internal ML platform for LINE, including a model serving and model management component using BentoML. I'm convinced that he will be a highly valuable member of our community and continue to make awesome contributions to the project.

Excited to have you onboard, Sungjun!

**Comment by yubozhao** [comment]
Welcome @withsmilo !

I really enjoy your suggestions and feedback. Your wealth of experience on MLOps is invaluable for this project. 

I can’t wait to work with you to improve BentoML. Let’s make BentoML the standard for packaging and deploying ML services!

**Reply by withsmilo** [reply]
Thanks, @yubozhao .
I am honored to be invited here as a core member. I promise to make some time to contribute to BentoML.
